src/components/button/button.js

```diff
@@ -0,0 +1,16 @@
+//@flow
+import {h} from 'preact';
+import {forwardRef} from 'preact/compat';
+
+const COMPONENT_NAME = 'Button';
```

Comment:
need to export it in `src/components/index.js`

Reply:
Not in src/components/index.js i think. in src/components/button/index.js

Reply:
Done